filament/src/materials/skybox.mat

```diff
@@ -23,8 +31,13 @@ material {
 fragment {
     void material(inout MaterialInputs material) {
         prepareMaterial(material);
-        vec3 sky = texture(materialParams_skybox, variable_eyeDirection.xyz).rgb;
-        sky *= frameUniforms.iblLuminance;
+        vec4 sky;
+        if (materialParams.constantColor) {
+            sky = materialParams.color;
+        } else {
+            sky = vec4(texture(materialParams_skybox, variable_eyeDirection.xyz).rgb, 1.0);
+        }
+        sky.rgb *= frameUniforms.iblLuminance;
```

Comment:
Should we ignore iblLuminance in constant color mode?

Reply:
I was wondering about that? Should we ignore, or just let the caller set it to 1?
I thought it would be nice to have the background color behave just like a contant skybox, especially we can draw the sun?

I was also wondering if we need a dedicated material, to avoid the test?

Reply:
okay I have decided to not multiply by iblLuminance.